packages/block-library/src/video/edit.native.js

```diff
@@ -229,7 +229,7 @@ class VideoEdit extends Component {
 						icon={ this.getIcon( ICON_TYPE.PLACEHOLDER ) }
 						onFocus={ this.props.onFocus }
 						autoOpenMediaUpload={
-							isSelected && ! src && wasBlockJustInserted
+							isSelected && wasBlockJustInserted
```

Comment:
I'm wondering if it's possible to have the case where `id` is not defined and `src` has a value 🤔, anyways this won't be a problem because it shouldn't affect the condition for auto-opening the picker.

Reply:
Indeed! That was a concern I noticed as well which prompted me to remove it altogether. Glad we are on the same page 💯